docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md

```diff
@@ -27,8 +27,17 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
-
-3. Make a call to the Kyma Environment Broker to create a Runtime on Azure. Find the list of possible request parameters [here](03-01-service-description.md).
+Alternatively, you can perform `kubectl port-forward` on the chosen pod to expose it on your local machine. Expose it to port `8080`:
+```bash
+  kubectl port-forward -n kcp-system deployments/kcp-kyma-environment-broker 8080
+```
+You should get information about the successful forwarding. For example:  
+```bash
+  Forwarding from 127.0.0.1:8080 -> 8080
+  Forwarding from [::1]:8080 -> 8080
+```  
+
+1. Make a call to the Kyma Environment Broker to create a Runtime on Azure. Find the list of possible request parameters [here](03-01-service-description.md).
```

Comment:
Why do we have step 1 after step 2? 
If it's part of this "alternative", then we should think about how to restructure this section, for example, put the whole section in two [toggle panels ](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/05-documentation-toggle.md)for two alternatives.